blog/12-20-22-Hackathons-101/Hackathons-101.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Hackathons 101- How These Events Can Transform Your Life and Tips for Winning
+description: In this article we will discuss the benefits of attending events, including networking, learning new skills, and gaining exposure to new ideas.
+authors:
+  - name: Ishan Pathak
+    title: Member, Celo Foundation
+    url: https://github.com/Ishanpathak1
+    image_url: https://github.com/ishanpathak1.png
+tags: [celosage, beginner]
+hide_table_of_contents: true
+slug: "/tutorials/hackathons-101-how-these-events-can-transform-your-life-and-tips-for-winning"
+---
+## Hackathons 101:How These events can Transform your Life and Tips for winning
```

Comment:
You can remove the title from here. When published we will have title at the top from frontmatter.
```suggestion
```